docker/Dockerfile.contrib

```
@@ -0,0 +1,25 @@
+FROM node:latest AS build
+USER root
+RUN mkdir -p /usr/local/lib/node_modules && chown node /usr/local/lib/node_modules
+USER node
+
+COPY --chown=node node-zwave-js /home/node/node-zwave-js
+COPY --chown=node zwavejs2mqtt /home/node/zwavejs2mqtt
+
+WORKDIR /home/node/node-zwave-js
+RUN npm install
+RUN npm run build
+RUN npm link
```

```suggestion
RUN npx lerna exec -- npm link
```

Afeter speaking with @AlCalzone seems this is better as will link all workspaces of zwavejs

---

@larstobi

---

@larstobi Is this working?